storage/integration_test.go

```diff
@@ -907,7 +932,7 @@ func TestIntegration_Autoclass(t *testing.T) {
 		if latestToggleTime.IsZero() {
 			t.Error("got a zero time value, want a populated value")
 		}
-		if latestToggleTime.Before(toggleTime) {
+		if !latestToggleTime.After(toggleTime) {
```

Comment:
Why was this condition check inverted? Isn't it the same thing, or is there some technicality which I am missing?

Reply:
They should not equal to each other either, hence the update

Reply:
I believe `time.Before` should return false if they are equal to each other; but anyway, I confirmed these 2 lines are the same thing.
Please ignore this as nitpick not really worth discussing.

Reply:
You're right they are the same, so we are catching the case if two toggle times are equal, thank you Brenna!